AlphaWallet/Tokens/ViewControllers/TokensViewController.swift

```diff
@@ -555,7 +555,7 @@ extension TokensViewController: UITableViewDataSource {
 
                 let didHideToken = strongSelf.viewModel.markTokenHidden(token: token)
                 if didHideToken {
-                    strongSelf.tableView.deleteRows(at: [indexPath], with: .automatic)
+                    strongSelf.tableView.deleteRows(at: indexPath.indexPathArrayForTokenType(), with: .left)
```

Comment:
Why `.left` and not `.automatic`?

Reply:
Automatic looks horrible when animated.

Reply:
I've switched it back to automatic. It only appears to be horrible only when you have a few entries.